### PR TITLE
Deprecate nightmare #dontmerge

### PR DESCRIPTION
**Pull request by eessex**
Follow up to #4956
- Removes all functionality and libraries related to Nightmare acceptance suite
- Renames new Cypress suite to use `acceptance` 

Related to https://artsyproduct.atlassian.net/browse/PLATFORM-2116

### CODE REVIEW

**Comment by eessex** [comment]
Unfortunately the acceptance tests seem to be blocked by the elasticsearch reindex.

File: src/test/acceptance/auth.js [hunk 1]
```diff
@@ -1,23 +0,0 @@
...
-    })
-  })
-
-  after(teardown)
-
-  xit("logs in", async done => {
-    await browser.page("/terms")
-    await browser.login()
-    const html = await browser.el(".main-layout-header-user")
-    html.should.containEql("Craig Spaeth")
```
Comment (anandaroop) [inline src/test/acceptance/auth.js]: 😢

File: test.config.js [hunk 2]
```diff
@@ -41,11 +41,6 @@ try {
   window.scrollTo = window.scrollTo || function() {}
 } catch (error) {}
 
-// Used for Acceptance tests
-require("raf/polyfill")
```
Comment (damassi) [inline test.config.js]: can this be removed from package.json?

Comment (eessex) [reply test.config.js]: Tests seem to be running fine without it, and I'm assuming based on comments that it was included specifically for acceptance tests.

Comment (eessex) [reply test.config.js]: Misread your comment - it is not in package json, seems to be a sub-dependency

**Comment by anandaroop** [comment]
See related discussion in https://artsy.slack.com/archives/CP9P4KR35/p1580164432042200

I'm putting a #dontmerge on this.

**Comment by damassi** [comment]
For the time being, can we remove the [job from circle](https://github.com/artsy/force/blob/master/.circleci/config.yml#L119-L120)?

**Comment by damassi** [comment]
Going to close this; feel free to reopen when work resumes

**Pull request closed without merging**